js/test/Exchange/test.account.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+// ----------------------------------------------------------------------------
+
+const assert = require ('assert')
+
+// ----------------------------------------------------------------------------
+
+module.exports = (exchange, account, method) => {
+
+    const format = {
```

Comment:
Let's use a transpileable formatting for this test, we will add it to other languages, see this example:

- https://github.com/ccxt/ccxt/blob/master/js/test/Exchange/test.market.js
- https://github.com/ccxt/ccxt/blob/master/build/transpile.js#L1602